### PR TITLE
Flakes

### PR DESCRIPTION
**Pull request by balsoft**
For flaky projects, we need to run in restricted mode and also have a `flake.nix`. This PR adds both.

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -216,7 +204,11 @@ in rec {
         # `npm prune` uses cache for some reason
         npm prune --production --cache=./npm-prune-cache/
         npm pack --ignore-scripts
+<<<<<<< Updated upstream
         ${untarAndWrap name [npmCmd]}
+=======
+        ${untarAndWrap "${pname}-${version}" [npmCmd]}
+>>>>>>> Stashed changes
```
Comment (kirelagin) [inline default.nix]: 🤔
Just be sure to squash it then :).

**Review by kirelagin — APPROVED** [review]
Ok, I admit that I have no idea what is happening in the filtering part, but if you tested it and it works, then I’m on board with the changes 😅.

(Just FTR, we do not strictly _need_ `flake.nix`, but, I guess, having it does not hurt.

**Comment by balsoft** [comment]
It's currently used in Ment, so yeah, i tested it :P

**Comment by balsoft** [comment]
Ok, since I'm going to use this in multiple projects, merging. If something breaks on update, we'll fix it then.

**Pull request merged**